tema14Ianuarie/src/com/sci/bv/tema14Ianuarie/AnimalProduct.java

```diff
@@ -0,0 +1,102 @@
+package com.sci.bv.tema14Ianuarie;
+
+
+/**
+ * In this Class are stocked animal products and their characteristics.
+ * (ID, quantity,name of product, list of vitamins, price, validity, weight)
+ */
+
+public class AnimalProduct {
+
+    private int ID;
+    private int Qty;
+    private String Name;
+    private double Temp;
+    private double Price;
+    private int Validity;
+    private double Weight;
```

Comment:
all attributes and all variables should start with a lower case letter. those are the coding guidelines. So you would have id, qty, name, temp, etc...